tensorcircuit/abstractcircuit.py

```diff
@@ -711,6 +712,12 @@ def from_qiskit(
         :type n: int
         :param inputs: possible input wavefunction for ``tc.Circuit``, defaults to None
         :type inputs: Optional[List[float]], optional
+        :param circuit_params: circuit attributes such as the number of qubits
```

Comment:
, keys of dict as given in ``Circuit.__init__`` construction function

Reply:
OK